code/modules/surgery/organs/tongue.dm

```diff
@@ -255,6 +255,19 @@
 /obj/item/organ/tongue/robot/handle_speech(datum/source, list/speech_args)
 	speech_args[SPEECH_SPANS] |= SPAN_ROBOT
 
+/obj/item/organ/tongue/honk
+	name = "clown voicebox"
+	desc = "A voice synthesizer that makes you 45% funnier."
+	slot = ORGAN_SLOT_TONGUE
+	icon_state = "tonguerobot"  // looks sufficiently tech-y enough to pass
+	say_mod = "honks"
+	attack_verb = list("honked", "clowns")
+	modifies_speech = TRUE
+	taste_sensitivity = 25 // not as good as an organic tongue
```

Comment:
Most of this stuff is copy and pasted off of the robot tongue, so make it a subtype of it

Reply:
Bad idea, don't do that - there's three fields from the robot tongue that are copied, but the robot tongue also adds a bunch of stuff like languages and emp reaction.